tests/cypress/support/commands.js

```diff
@@ -231,6 +231,19 @@ Cypress.Commands.add(
                 let target = Math.max(targetTopic, targetLayers)
                 // If a layer has been set via adminId we just increment by one.
                 target += Boolean(queryParams.adminId)
+                // When handling a legacy parameter, the bod Layer Id might add extra
+                // layers, thus the need to check if those extra layers have been added
+                if (!withHash && active !== target) {
+                    let targetBodIdParameter = 0
+                    state.layers.activeLayers.forEach((layer) => {
+                        targetBodIdParameter += Boolean(
+                            layer.geoAdminID in queryParams &&
+                                !queryParams.layers?.includes(layer.geoAdminID) &&
+                                !getters.currentTopic?.layersToActivate.includes(layer.geoAdminID)
+                        )
+                    })
+                    target = +targetBodIdParameter
```

Comment:
I suspect you wanted to write `target += targetBodIdParameter`? `= +` seems a bit off
You might want to cover this with a test, or expand the test you already wrote to cover this if block thoroughly. I bet this line breaks the logic you wanted to implement